.travis.yml

```diff
@@ -85,6 +87,7 @@ script:
       docker logout;
     fi;
   fi;
+- docker run --rm -t -e GITHUB_TOKEN=$ASTRONOMER_TOKEN -v "$HOME/.cache/astronomer_report:/data" ullaakut/astronomer iov-one/weave
```

Comment:
pp: do not run "untrusted" containers during the build process. If this is an important feature just rebuild the container and publish to iov1 repo.

Reply:
Ofc thanks for the warning!

Reply:
This is just a repo analyzing tool that I played around and I did not have any intention to merge it.